apps/web/components/walkthrough/Walkthrough.tsx

```diff
@@ -48,7 +49,6 @@ const Walkthrough = observer((): JSX.Element => {
 
   return (
     <div className="web-Walkthrough" data-testid={TESTID_WALKTHROUGH}>
```

Comment:
Could we use Html5 semantics instead of nested <divs> ?

Reply:
Good call. I hadn't paid attention to the overall structure in a while. Moved the step tracker `<section>` up into the walkthrough overall and made the `web-Walkthrough--Content` it's own `<section>`. I left this overall wrapper as a `<div>` because it's just there for styling, this component is already wrapped in a `<main>` element from the root layout and I don't think there is a better way to structure that. Thoughts?